### DISCUSSION TITLE
Collections for "well-known" APIs

### DISCUSSION THREAD

**Discussion by micro-jumbo** [usebruno/bruno#2089] [General]
Hey,

I'm new to bruno, so maybe it exists somewhere already, but I didn't find it.
There are tons of publicly available APIs on the web and lots of people are trying to integrate with them (like Stripe, each AWS service exposing HTTP endpoint, etc...). 
I've been working with AWS APIs recently and I had to manually define all requests I needed. It would be great if I could just import collection for existing AWS service and use it locally. Is there any repo that stores definitions for these "well-known" APIs?

Thanks

**Comment by rreyn-bruno** [comment]
Hi @micro-jumbo - this is something we'll be taking up within the next couple of months. Part of it is encouraging companies who provide other collections (e.g - Postman), to also provide Bruno collections. 

That said, as a workaround, you can always find a Postman collection if available and import it into Bruno :).
